src/parser/mod.rs

```diff
@@ -3609,6 +3609,16 @@ impl<'a> Parser<'a> {
                         self.expected("IN or BETWEEN after NOT", self.peek_token())
                     }
                 }
+                Keyword::MEMBER => {
+                    if self.parse_keyword(Keyword::OF) {
+                        let _ = self.expect_token(&Token::LParen);
+                        let expr2 = self.parse_expr()?;
+                        let _ = self.expect_token(&Token::RParen);
```

Comment:
It looks like we're dropping potential error returned by the expect_token calls (we should probably add a test case for this behavior)?